qa/tasks/cephfs/test_misc.py

```diff
@@ -223,6 +223,27 @@ def test_dump_inode_hexademical(self):
         info = self.fs.mds_asok(['dump', 'inode', hex(ino)])
         assert info['path'] == "/foo"
 
+    def test_dump_dir(self):
+        self.mount_a.run_shell(["mkdir", "-p", "foo/bar"])
+        dirs = self.fs.mds_asok(['dump', 'dir', '/foo'])
+        assert type(dirs) is list
+        for dir in dirs:
+            assert dir['path'] == "/foo"
```

Comment:
as venky mentioned, using self.assert* would be better, do try to replace all assertions if possible, for e.g you can use `self.assertEquals()` here

Reply:
> as venky mentioned, using self.assert* would be better, do try to replace all assertions if possible, for e.g you can use `self.assertEquals()` here

OK! I have replaced all assertions in this file.

Reply:
thanks! but i guess you got me a little bit wrong, by all i meant those in your test case. However if you feel you want to replace them as well, maybe that can be done separately in another commit, but i'm not very sure about it.